### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by dantarasov**

### CODE REVIEW

File: ui/homework/homework_1/test1.ts [hunk 1]
```diff
@@ -4,6 +4,9 @@
 */
 
 type Person = {
+  name?: string;
+  age?: number | unknown;
```
Comment (aleksandrashmoylova) [inline ui/homework/homework_1/test1.ts]: are you sure about this type?

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,2 @@
-node_modules
+3node_modules
```
Comment (aleksandrashmoylova) [inline .gitignore]: why did you change it?

Comment (dantarasov) [reply .gitignore]: accidentally. I wasn't supposed to do that.

**Review by aleksandrashmoylova — APPROVED** [review]
